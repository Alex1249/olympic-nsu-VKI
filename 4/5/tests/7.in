84 73
493 -291 36 -343 172 744 -625 -792 181 -226 -332 -740 18 -31 290 404 881 90 -618 277 586 856 -935 62 429 -669 345 477 -753 296 979 218 -166 179 -407 186 -372 244 -830 -110 810 735 688 961 -144 -50 818 -238 -933 -999 243 639 171 533 -454 7 -445 495 235 -657 56 236 365 -46 -375 -789 -262 228 -765 290 -783 -82 -207 198 456 -820 263 974 -246 717 203 270 -504 104
1 771
1 1604
0 41
1 414
1 1423
1 2163
1 2009
1 1835
0 838
1 25
0 996
1 81
0 1280
0 1978
0 97
1 2532
0 1206
0 2331
1 2252
0 1468
0 772
1 583
0 133
1 54
1 2606
1 2445
1 1998
0 691
0 408
1 1792
0 465
0 1943
1 1868
0 1014
0 2584
0 2251
0 1784
0 777
1 2579
1 1212
1 2413
1 1690
1 1942
1 218
1 1186
1 457
1 1496
1 154
1 687
1 1533
0 1270
0 1524
1 466
0 1122
1 237
0 229
1 365
1 870
0 675
1 2287
1 5
0 188
0 2527
0 2580
0 1605
1 474
0 972
1 2399
0 506
1 2298
1 890
1 2267
0 1044
