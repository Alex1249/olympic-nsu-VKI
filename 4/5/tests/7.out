67109357 -4194595 16420 -1073742167 172 736 -593 -67109656 181 -226 -332 -740 18 -31 4202786 404 881 90 -618 277 586 66392 -935 62 268435885 -669 345 33554941 -753 296 979 218 -2214 179 -407 186 -372 536871164 -830 -622 810 735 688 961 -144 -58 946 -2282 -933 -999 134217971 639 171 533 -454 7 -445 1049071 524523 -657 568 204 131437 -46 -375 -789 -262 4324 -765 290 -1049359 -82 -223 198 457 -820 262407 974 -246 134218445 4171 131342 -504 104 
